pallets/msa/src/tests.rs

```diff
@@ -437,36 +437,6 @@ fn test_retire_msa_success() {
 	})
 }
 
-#[test]
```

Comment:
We have tests for the MSA pallet signed extensions already, these can simply be converted to a signed extension test and unit tests for `validate_msa_retirement` and would be better than deleting them.

Reply:
Agreed. I will update.